internal/app/cache/cache.go

```diff
@@ -33,14 +33,14 @@ type cache struct {
 
 // Response stores the raw Discovery Response alongside its marshaled resources.
 type Response struct {
-	raw                v2.DiscoveryResponse
-	marshaledResources []*any.Any
+	Raw                v2.DiscoveryResponse
+	MarshaledResources []gcp_types.MarshaledResource
 }
 
 type Resource struct {
-	resp           *Response
-	requests       []*v2.DiscoveryRequest
-	expirationTime time.Time
+	Resp           *Response
+	Requests       []*v2.DiscoveryRequest
+	ExpirationTime time.Time
```

Comment:
Does ExpirationTime need to be exported?

Reply:
Not for the orchestrator, but I'm sure that we'll need it for our admin APIs later when we dump what's in the cache.

Reply:
makes sense!